src/ducks/Studio/Alerts/suggestions/price.js

```diff
@@ -3,36 +3,43 @@ import { createSuggestion } from './helpers'
 import Value from '../Value'
 import { PRICE_CHANGE_TYPES } from '../../../Signals/utils/constants'
 import {
-  buildPriceSignal,
-  buildPricePercentUpDownSignal
+  buildValueChangeSignal,
+  buildPercentUpDownSignal
 } from '../../../Signals/utils/utils'
 import { Metric } from '../../../dataHub/metrics'
 
-const { formatter } = Metric.price_usd
+export const SIGNAL_BELOW = 'BELOW'
+export const SIGNAL_ABOVE = 'ABOVE'
+export const VALUE_IFS = ['drops below', 'rises above']
 
-const SIGNAL_BELOW = 'BELOW'
-const SIGNAL_ABOVE = 'ABOVE'
-const PRICE_IFS = ['drops below', 'rises above']
+export const suggestValueChange = metric => {
```

Comment:
Now the name`suggestValueChange` breaks the semantic of the function, because it no longer returns a **suggestion**, but a suggester (a function), that will return a suggestion. My suggestion is to rename it to something like `buildValueChangeSuggester`